pkg/controller/pxc/controller.go

```diff
@@ -603,6 +604,11 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 	err = r.client.Create(context.TODO(), nodeSet)
 	if err != nil && !k8serrors.IsAlreadyExists(err) {
 		return errors.Wrap(err, "create newStatefulSetNode")
+	} else if k8serrors.IsAlreadyExists(err) {
+		err := r.client.Update(context.TODO(), nodeSet)
```

Comment:
this one is very tricky part, because it will cause update to statefulset each reconcile loop.
Let's use our special created function for this

Reply:
This logic was here for ages. This task is about fixing labels.
If we want to fix this let's do it in PR

Reply:
ok. Then revert this change

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/803/commits/a3c813e6f96f63baaf75e23c89f22b48cb4afc43